packages/eslint-plugin/tests/rules/import-order.test.ts

```diff
@@ -0,0 +1,63 @@
+import rule from '../../src/rules/import-order';
+import { RuleTester } from '../RuleTester';
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: {}
+  },
+  parser: '@typescript-eslint/parser'
+});
+
+ruleTester.run('import-order', rule, {
+  valid: [
+    "import { a, b } from 'foo'",
+    "import { A, b } from 'foo'",
+    "import { a, B } from 'foo'",
+    "import { A, B } from 'foo'",
+    "import { a, _b } from 'foo'",
+    "import * as bar from 'foo'",
+    `import b from 'a';
+     import a from 'b';
+    `,
+    `import a from 'a';
+     import { b, c } from 'bc';
+     import d from 'd';
+    `
```

Comment:
can you add some test cases for import equal?
```
import foo = ....
```

https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require